lib/Clients/Service/Localise.php

```diff
@@ -66,7 +76,29 @@ public function export( array $args ) : string {
 			throw new Exception( 'Authenticate should be called first' );
 		}
 
-		return $this->client->exportLocale( $args );
+		$url = sprintf(
+			'https://localise.biz/api/export/locale/%s.%s',
```

Comment:
Após o base_uri actualizas este também.

```suggestion
			'export/locale/%s.%s',
```